src/main/java/org/multi/routes/controller/Controller.java

```diff
@@ -0,0 +1,72 @@
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import static org.apache.logging.log4j.Level.*;
+
+public class Controller {
+    private final static Logger logger = LogManager.getLogger(Controller.class);
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        ExecutorService executorService = Executors.newFixedThreadPool(3);
+        DataEntityInitializerImpl dataEntityInitializerImpl = DataEntityInitializerImpl.getInstance();
+
+        BusStopService busStopService = new BusStopServiceImpl();
+        BusService busService = new BusServiceImpl();
+        BusRouteService busRouteService = new BusRouteServiceImpl();
+        PassengerService passengerService = new PassengerServiceImpl();
+
+        busStopService.setBusStopRepository(new BusStopsRepositoryImpl(dataEntityInitializerImpl));
+        busService.setBusRepository(new BusRepositoryImpl(dataEntityInitializerImpl));
+        busRouteService.setBusRouteRepository(new BusRouteRepositoryImpl(dataEntityInitializerImpl));
+        passengerService.setPassengerRepository(new PassengerRepositoryImpl(dataEntityInitializerImpl));
+
+        List<Bus> buses = busService.getBuses();
+        List<Passenger> passengers = passengerService.getPassengers();
+        List<BusStop> stops = busStopService.getBusStops();
+        List<BusRoute> busRoutes = busRouteService.getBusRoutes();
+
+
+        busRoutes.forEach(r -> logger.log(INFO, r));
+        List<Future<String>> futures = new ArrayList<>();
+
+        for (Bus bus : buses) {
+            futures.add(executorService.submit(bus));
+        }
+        for (Future<String> future : futures) {
+            logger.log(INFO, future.get());
+        }
+        executorService.shutdown();
+
+        passengers.forEach(p -> logger.log(INFO, p + " is arrived: " + p.isArrivedAtDestination()));
+
+        List<Passenger> notArrivedPassengers = passengers.stream().filter(p -> !p.isArrivedAtDestination()).toList();
+
+
+        buses.forEach(bus -> logger.log(INFO, bus + " passengers " + bus.getPassengers()));
+        stops.forEach(stop -> logger.log(INFO, stop + " " + stop.getPassengerLine()));
+
+        busService.getBuses().forEach(entity -> logger.log(INFO, entity.getId()));
+        busStopService.getBusStops().forEach(entity -> logger.log(INFO, entity.getId()));
+        busRouteService.getBusRoutes().forEach(entity -> logger.log(INFO, entity.getId()));
+        passengerService.getPassengers().forEach(entity -> logger.log(INFO, entity.getId()));
+        passengers.forEach(p -> logger.log(INFO, p + " is arrived: " + p.isArrivedAtDestination()));
+        notArrivedPassengers.forEach(p -> logger.log(ERROR, p + " Current stop" + p.getCurrentStop()
+                + "\nDestination: " + p.getDestination()));
+    }
```

Comment:
in accordance with Java code convention we should use each lambda function from the new line. Even if we think that one-line writing looks more laconic, nevertheless, each new function inside a lambda must start on a new line.

I don't see much point in redoing it here. It's not a mistake, just keep it in mind in the future.